captain/utils/nodes_metadata.py

```diff
@@ -0,0 +1,46 @@
+import fnmatch
+import json
+import os
+
+nodes_dir = "PYTHON/nodes"
+
+# The pattern to match for Python files
+pattern = "*.py"
+
+badbadnotgood = ["VCTR.py", "__init__.py", ".DS_Store"]
```

Comment:
why is VCTR.py in this list, seems like it is not something "harmful"?

Reply:
Actually, we can remove that.